### ISSUE TITLE
XML Comments moved to first column

### ISSUE THREAD

**Issue by m1ga** [Glavin001/atom-beautify#409]
in a XML file the ceomments are moved to the first column:

```
            <View class="btn_left" id="btn_back">
                <Label text="Zurück"/>
            </View>
            <!-- 3 views -->
            <View class="btn_left" id="btn_back">
                <Label text="Zurück"/>
            </View>
```

is translated to

```
            <View class="btn_left" id="btn_back">
                <Label text="Zurück"/>
            </View>
<!-- 3 views -->
            <View class="btn_left" id="btn_back">
                <Label text="Zurück"/>
            </View>
```

after beautifying.

**Comment by jmenges** [comment]
That is caused by the package that handles xml by default, which is called prettydiff. You might want to report this on their github (https://github.com/prettydiff/prettydiff).
I verifyied the problem directly. For the time beeing you could switch to js-beautify for xml in the settings.
```
%prettydiff source:"test.xml" mode:"beautify"
<test>
  <test2>
    <View class="btn_left" id="btn_back">
      <Label text="Zurück"/>
    </View>
<!-- 3 views -->
    <View class="btn_left" id="btn_back">
      <Label text="Zurück"/>
    </View>
  </test2> that worked for me.

</test>
```

**Comment by m1ga** [comment]
ah ok, sure. Didn't know that prettydiff is also used for xmls

**Comment by Glavin001** [comment]
/cc @prettydiff

Thank you @jmenges for helping out.

@m1ga , if you would like to try using `JS Beautify` for `XML` language, as @jmenges suggested, you can select it in the Atom Beautify package settings with the option `Language Config - XML - Default Beautifier`. See https://github.com/Glavin001/atom-beautify/blob/master/docs/options.md#language-config---xml---default-beautifier for more details.

**Comment by prettydiff** [comment]
Hold off on creating a bug. I am about to fix this in just a few minutes.

**Comment by Glavin001** [comment]
Awesome, thanks @prettydiff!

**Comment by m1ga** [comment]
@prettydiff thank you!

**Comment by m1ga** [comment]
That was quick! Just tried it online and its fixed in prettydiff 1.12.1

**Comment by Glavin001** [comment]
You can uninstall and reinstall Atom Beautify to trigger Pretty Diff dependency to be updated.

**Comment by dancvogel** [comment]
I'm experiencing the same issue on .jsx files.

**Comment by prettydiff** [comment]
@dancvogel Can you paste a small sample code here so that I may reproduce the problem?

**Comment by dancvogel** [comment]
@prettydiff 
```
App.components.app.frame.navbar = React.createClass({
    ///---------------------------------------------------------------
    ///
    ///	@description
    ///		React render function.
    ///
    ///--------------------------------------------------------------
    render: function() {
        return <App.bootstrap.Navbar brand='React-Bootstrap' inverse toggleNavKey={0}>
                <App.bootstrap.CollapsibleNav eventKey={0}>
                    <App.bootstrap.Nav navbar>
                        <App.bootstrap.NavItem className="page1" eventKey={1} href='#page1'>Translations</App.bootstrap.NavItem>
                        <App.bootstrap.NavItem className="page2" eventKey={2} href='#page2'>Inheritance</App.bootstrap.NavItem>
                    </App.bootstrap.Nav>
                </App.bootstrap.CollapsibleNav>
            </App.bootstrap.Navbar>;
    }
});
```

**Comment by prettydiff** [comment]
When I tried in the online Pretty Diff tool it formatted correctly for me.  The automatic language detection was having some trouble with this code sample so I just dropped a `var a = 1;` at the top of the sample when I entered it in the tool.  The automatic language detection determined it is React JSX and I did not notice any problems.

JSX also allows JS comments to be embedded within the XML tags.  I do claim to support this, but it is kind of an edge case, so if you notice any problems in this area just let know with a code sample and I will fix it up.
